src/app/content/highlights/components/SummaryPopup/HighlightsHelpInfo.tsx

```diff
@@ -0,0 +1,98 @@
+import * as Cookies from 'js-cookie';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import styled, { css, keyframes } from 'styled-components';
+import { Times } from 'styled-icons/fa-solid/Times/Times';
+import { useAnalyticsEvent } from '../../../../../helpers/analytics';
+import { user as userSelector } from '../../../../auth/selectors';
+import { PlainButton } from '../../../../components/Button';
+import htmlMessage from '../../../../components/htmlMessage';
+import { bodyCopyRegularStyle } from '../../../../components/Typography';
+import theme from '../../../../theme';
+
+// This is copied from CallToActionPopup > styles.tsx
+// Wher should we store this kind of functions?
+const slideInFromBottom = keyframes`
+  0% {
+    bottom: -100%;
+  }
+
+  100% {
+    bottom: 0;
+  }
+`;
+
+const slideInAnimation = css`
+  animation: ${800}ms ${slideInFromBottom} ease-out;
+`;
+
+// tslint:disable-next-line: variable-name
+const Wrapper = styled.div`
+  position: fixed;
+  bottom: 0;
+  z-index: ${theme.zIndex.highlightsHelpInfoMobile};
+  ${bodyCopyRegularStyle}
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 6rem;
+  padding: 0 2rem;
+  background-color: ${theme.color.neutral.formBackground};
+  border: 1px solid ${theme.color.neutral.formBorder};
+  ${slideInAnimation}
+
+  a {
+    text-decoration: none;
```

Comment:
i don't think this is good, but i see that it is like this in the design, wait to see what devika says https://openstax.slack.com/archives/C8V1XET1R/p1581603867240200